cdatransform/transform/main.py

```diff
@@ -30,8 +26,8 @@ def filter_cases(reader, case_list):
             break
         else:
             if case.get("id") in cases:
-                cases.pop(case.get("id"))
-                yield case
+                cases.remove(case.get("id"))
+            yield case
```

Comment:
@pshapiro4broad there is a bug introduced here. We should eventually have a test to cover this.

The yield should be executed under the `if`. The indent has been lost in line 30.  Without the indent we will get the case returned each loop. Welcome to Python!

Reply:
I'll have to check my IDE settings, this change didn't show up in my local diff. Most likely I have a setting enabled to hide whitespace only changes.

And yes having a test would've been good here!